plugins/woocommerce/src/Internal/Admin/Settings/PaymentsProviders.php

```diff
@@ -580,7 +626,13 @@ function ( $suggestion ) use ( $preferred_psp, $preferred_apm ) {
 	 * @return ?array The payment extension suggestion details, or null if not found.
 	 */
 	public function get_extension_suggestion_by_id( string $id ): ?array {
-		return $this->extension_suggestions->get_by_id( $id );
+		$suggestion = $this->extension_suggestions->get_by_id( $id );
```

Comment:
To ensure consistency, always enhance the suggestion data, the same way to provide it in the providers list.